code/game/machinery/vending.dm

```diff
@@ -3356,7 +3357,8 @@ var/global/num_vending_terminals = 1
 		/obj/item/weapon/dice/loaded = 15,
 		/obj/item/weapon/dice/loaded/d20 = 15,
 		/obj/item/weapon/skull = 20,
-		/obj/item/weapon/storage/bag/wiz_cards/frog = 20
+		/obj/item/weapon/storage/bag/wiz_cards/frog = 20,
+		/obj/item/weapon/melee/rsscimmy = 25000
```

Comment:
Rune scimitar has a base value of 25,600 according to the OSRS wiki.

https://oldschool.runescape.wiki/w/Rune_scimitar